packages/documentation-site/patternfly-docs/content/PatternFly-AI/about-generative-ui.md

```diff
@@ -0,0 +1,73 @@
+---
+id: About generative UIs
+section: PatternFly-AI
+subsection: Generative UIs
+---
+
+# PatternFly + Generative UIs
+
+## What is generative UI?
+
+Generative UI is an emerging approach to user interface design that leverages artificial intelligence to dynamically create and adapt interface elements based on context, user needs, and data. Unlike traditional static interfaces, generative UIs can produce layouts, components, and visual styles in real-time, offering more flexible and personalized user experiences.
+
+## PatternFly's exploration: Compass
+
+Generative UI represents a significant opportunity for PatternFly to explore new patterns, layouts, and styles that support AI-driven interface generation. PatternFly has been calling this proof of concept Compass. It investigates how the design system can evolve to support generative UI use cases.
+
+**Important**: Compass is best suited for use as a POC in other proof-of-concept generative UI use cases. It is **not yet production quality code** and should be used for exploration and experimentation purposes only.
+
+### AI-enabled seed app
+
+We are creating an [AI-enabled seed app](https://github.com/gitdallas/patternfly-react-seed/tree/compass_theme) that you can use to create your own PatternFly Compass-based POC. The seed app includes the Compass layout fully wired out with all styles set up, providing a solid foundation that you can build on top of. It can be leveraged with AI tools like Cursor or Claude Code to build out your own POC, making it easier to experiment with generative UI patterns and explore new use cases.
+
+### What Compass includes
+
+The Compass POC introduces several new capabilities:
+
```

Comment:
```suggestion

| Feature | Description | Usage |
| --- | --- | --- | 
| Design token | glass-tokens | Used to apply glass-like visuals to .. |
| Design token | gradient-tokens | Used to apply gradients to backgrounds and borders.|
| Layout | Compass layout | A UI layout that places controls above, below, and on both sides of a primary content area. |
```
just jotted down a loose thing, but do you think it would be helpful to contain the new features in something similar to this, or would it be more confusing? 

I would place the demos line above this table for easier access: "To explore Compass and its features, explore our full-page demo with card view and table layouts.